.gitignore

```diff
@@ -40,4 +40,6 @@ oprofile*
 *.log
 position.txt
 *.9
-*.glade.h
```

Comment:
Why did you remove this line?

Reply:
i will try to remember, it is connected to our gui i think

Reply:
This is one of the reasons to write good commit messages ;-)